.circleci/config.yml

```diff
@@ -1,248 +1,131 @@
+# https://app.circleci.com/pipelines/github/prodigyjoe/test-urban-circleci-site
+
+# ## General Environment Variables set in CircleCI
+
+# * TERMINUS_TOKEN created in https://dashboard.pantheon.io/users/#account/tokens/
+# * DEFAULT_BRANCH set to `main` to ensure our primary branch delivers to dev
+
+# ## Setup Details
+
+# * Review https://pantheon.io/docs/drupal-9#create-a-new-drupal-9-site-with-integrated-composer
+# * Create new site from Drupal 9 upstream: https://dashboard.pantheon.io/sites/create?upstream_id=e96c6794-77fe-4931-9a20-48a2fe1a3789
+# * Clone Pantheon locally, update pantheon.yml to include `build_step: false`
+# * Copy `pantheon.yml` and `pantheon.upstream.yml` into your own repo
+# * Ensure the above environment variables in CircleCI
+
+workflows: #Defines a list of jobs and their run order
   version: 2
-  build_deploy_and_test:
-    jobs:
-      - configure_env_vars
-      - static_tests
-      - deploy_to_pantheon:
-          requires:
-            - static_tests
-      - visual_regression_test:
-          requires:
-            - configure_env_vars
-            - deploy_to_pantheon
-          filters:
-            branches:
-              ignore:
-                - master
-      - behat_test:
-          requires:
-            - deploy_to_pantheon
-            - visual_regression_test
-
-  scheduled_update_check:
-    triggers:
-       - schedule:
-           cron: "0 0 * * *"
-           filters:
-             branches:
-              only:
-                - master
-    jobs:
-      - composer_lock_updater
+  build_and_push: #name of workflow(runs job named build and job named push)
+      jobs: #A collection of steps, which run commands/scripts as required. Must declare an executor that is either docker, machine, windows, or macos.
+        - build #Run job called build
+        - pantheon/push:  #Run pantheon orb's job named push
+            requires: #Require a dependency by job name
+              - build #Require a job named build
+            checkout: false #Set pantheon orb's push job's paramater checkout to false
+            terminus_clone_env: live  #Set pantheon orb's push job's paramater terminus_clone_env to live
+            clone_content: false  #Set pantheon orb's push job's paramater clone_content to false
+            env_create_max_time: "15m"  #Set pantheon orb's push job's paramater env_create_max_time to 15 minutes
+            pre-steps:  #Steps executed before any other step in the job. Optional special argument for jobs
+             - checkout #Run pantheon orb's job named checkout
+             - attach_workspace: # ?
+                at: .   # ?
+             - run: #Invoke all command-line programs
+                name: Replace repo .gitignore so built codebase makes it to Pantheon  #Title of the step to be shown in the CircleCI UI
+                command: |  #Run the following commands via shell
+                  rm -vf .gitignore
+                  mv -v pantheon.gitignore .gitignore
```

Comment:
Do you have this file in your project directory? You may get an error if you don't have it.

Reply:
The [current config file](https://github.com/prodigyjoe/d9-circleci-site/blob/master/.circleci/config.yml) doesn't have the changes yet, but it will once it's merged